src/Groups/GAPGroups.jl

```diff
@@ -102,7 +102,28 @@ positive integer `n` such that `x^n` is the identity of `G`.
 For a group `x`, the order of `x` is the number of elements in `x`.
 
 An exception is thrown if the order of `x` is infinite,
-use [`isfinite`](@ref) in order to check for finiteness.
+use [`isfinite`](@ref) for checking the finiteness of a group,
+and [`is_finiteorder`](@ref) for checking whether a group element
+has finite order.
+
+# Examples
+```jldoctest
+julia> g = symmetric_group(3);
+
+julia> order(g)
+6
+
+julia> order(gen(g, 1))
+3
+
+julia> g = free_group(1);
+
+julia> isfinite(g)
```

Comment:
Meta-question: should we say `is_finite` here, to be more consistent with the rest? (I have no strong opinion on this, just noticed the contrast to the next input line)